tests/test_ffmpeg.py::test_writer_pixelformat_size_verbose - AssertionError: assert 'yuv420p' == 'yuv420p(progressive)' (on Gentoo)

My educated guess would be that this is because we have a different ffmpeg version than used on the CI.

```pytb
________________________________________________ test_writer_pixelformat_size_verbose _________________________________________________

tmpdir = local('/tmp/portage/dev-python/imageio-2.31.0/temp/pytest-of-portage/pytest-0/test_writer_pixelformat_size_v0')

    def test_writer_pixelformat_size_verbose(tmpdir):
        # Check that video pixel format and size get written as expected.
    
        # Make sure verbose option works and that default pixelformat is yuv420p
        tmpf = tmpdir.join("test.mp4")
        W = iio.get_writer(str(tmpf), ffmpeg_log_level="warning")
        nframes = 4  # Number of frames in video
        for i in range(nframes):
            # Use size divisible by 16 or it gets changed.
            W.append_data(np.zeros((64, 64, 3), np.uint8))
        W.close()
    
        # Check that video is correct size & default output video pixel format
        # is correct
        W = iio.get_reader(str(tmpf))
        assert W.count_frames() == nframes
        assert W._meta["size"] == (64, 64)
>       assert "yuv420p" == W._meta["pix_fmt"]
E       AssertionError: assert 'yuv420p' == 'yuv420p(progressive)'
E         - yuv420p(progressive)
E         + yuv420p

W          = <imageio.plugins.ffmpeg.FfmpegFormat.Reader object at 0x7fe2c4bc70a0>
i          = 3
nframes    = 4
tmpdir     = local('/tmp/portage/dev-python/imageio-2.31.0/temp/pytest-of-portage/pytest-0/test_writer_pixelformat_size_v0')
tmpf       = local('/tmp/portage/dev-python/imageio-2.31.0/temp/pytest-of-portage/pytest-0/test_writer_pixelformat_size_v0/test.mp4')

tests/test_ffmpeg.py:386: AssertionError
```

Would be it be OK to make it do something like `W._meta["pix_fmt"] in ("yuv420p", "yuv420p(progressive)")`?

imageio 2.31.0
imageio-ffmpeg 0.4.8
Pyhon 3.10.11

--- a/tests/test_ffmpeg.py
+++ b/tests/test_ffmpeg.py
@@ -383,7 +383,7 @@ def test_writer_pixelformat_size_verbose(tmpdir):
     W = iio.get_reader(str(tmpf))
     assert W.count_frames() == nframes
     assert W._meta["size"] == (64, 64)
-    assert "yuv420p" == W._meta["pix_fmt"]
+    assert W._meta["pix_fmt"] in ("yuv420p", "yuv420p(progressive)")
 
     # Now check that macroblock size gets turned off if requested
     W = iio.get_writer(str(tmpf), macro_block_size=1, ffmpeg_log_level="warning")
@@ -393,7 +393,7 @@ def test_writer_pixelformat_size_verbose(tmpdir):
     W = iio.get_reader(str(tmpf))
     assert W.count_frames() == nframes
     assert W._meta["size"] == (106, 100)
-    assert "yuv420p" == W._meta["pix_fmt"]
+    assert W._meta["pix_fmt"] in ("yuv420p", "yuv420p(progressive)")
 
     # Now double check values different than default work
     W = iio.get_writer(str(tmpf), macro_block_size=4, ffmpeg_log_level="warning")
@@ -403,7 +403,7 @@ def test_writer_pixelformat_size_verbose(tmpdir):
     W = iio.get_reader(str(tmpf))
     assert W.count_frames() == nframes
     assert W._meta["size"] == (68, 64)
-    assert "yuv420p" == W._meta["pix_fmt"]
+    assert W._meta["pix_fmt"] in ("yuv420p", "yuv420p(progressive)")
 
     # Now check that the macroblock works as expected for the default of 16
     W = iio.get_writer(str(tmpf), ffmpeg_log_level="debug")
@@ -414,7 +414,7 @@ def test_writer_pixelformat_size_verbose(tmpdir):
     assert W.count_frames() == nframes
     # Check for warning message with macroblock
     assert W._meta["size"] == (144, 112)
-    assert "yuv420p" == W._meta["pix_fmt"]
+    assert W._meta["pix_fmt"] in ("yuv420p", "yuv420p(progressive)")
 
 
 def test_writer_ffmpeg_params(tmpdir):

